woocommerce/woocommerce-ios
# Update project to Swift 5

Closes #819. 

It turns out that you *don't* have to convert all the pods to Swift 5 compatibility in order to convert the main project 🎉 

## To Test
- run `bundle exec pod install`
- run unit tests
- build and run the app

This warning may still appear:
⚠️ Conversion to Swift 5 is available

if you click on it, you will notice that it's for several of the Pods and *not* the main project.
<img width="400" alt="Screen Shot 2019-04-26 at 11 02 48 AM" src="https://user-images.githubusercontent.com/1062444/56820973-df813e00-6812-11e9-9be7-4dd50b9aa362.png">


Update release notes:
